src/Fonts/fonts.style.ts

```diff
@@ -0,0 +1,120 @@
+import { createGlobalStyle, css } from "styled-components";
+import DefaultFont from "./DefaultFont";
+import ShadowStylesFonts from "./ShadowStyleFonts";
+
+import ie9Compat00 from "../../public/fonts/TTNorms/35C396_0_0.eot";
```

Comment:
Nitpick, but it looks like lots of legacy stuff I hope we can get rid of some

Reply:
Yes, I just converted it and get it to work with the ShadowDOM. Also I wasn't aware at that moment that the library is full of deprecated legacy code. Maybe it is a job for someone who know's whats been used and what not to get rid of it.